forui/lib/src/widgets/breadcrumb.dart

```diff
@@ -0,0 +1,222 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:forui/forui.dart';
+import 'package:meta/meta.dart';
+
+/// A Breadcrumb.
+///
+/// Displays the path to the current resource using a hierarchy of links..
+///
+/// See:
+/// * https://forui.dev/docs/navigation/breadcrumb for working examples.
+/// * [FBreadcrumbStyle] for customizing a breadcrumb's appearance.
+final class FBreadcrumb extends StatelessWidget {
+  /// The breadcrumb's style. Defaults to the appropriate style in [FThemeData.breadcrumbStyle].
+  final FBreadcrumbStyle? style;
+
+  /// The items.
+  final List<FBreadcrumbItem> items;
+
+  final Widget? dividerIcon;
+
+  /// Creates an [FBreadcrumb].
+  const FBreadcrumb({
+    required this.items,
+    this.dividerIcon,
+    this.style,
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final style = this.style ?? context.theme.breadcrumbStyle;
+    final dividerIcon =
+        this.dividerIcon ?? FIcon(FAssets.icons.chevronRight, color: Colors.white, size: style.iconSize);
+
+    return Row(
+      children: [
+        for (final (index, item) in items.indexed) ...[
+          if (item is _Crumb) item else item as _NestedCrumbs,
+          if (index < items.length-1) Padding(padding: const EdgeInsets.symmetric(horizontal: 5), child: dividerIcon),
+        ]
+      ],
+    );
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty('style', style));
+  }
+}
+
+abstract class FBreadcrumbItem {
+  const FBreadcrumbItem._();
+
+  /// Creates a crumb.
+  ///
+  /// It is typically used on pages at the root of the navigation stack.
```

Comment:
Is this statement correct? Example: "A > B > C", C isn't technically "at the root of the navigation stack".